src/Spectre.libGaussianMixtureModelling/GaussianMixtureModel.h

```diff
@@ -20,6 +20,7 @@
 #pragma once
 #include <vector>
 #include <gsl.h>
+#include "../Common/DataTypes.h"
```

Comment:
Should be w.r.t. root of solution.

Reply:
I tried doing that, but for some reason the IDE said that it cannot find the path. Now I've found it to compile just fine.

Reply:
Then, probably property sheet is not added so you are missing `$(SolutionDir)` in `Addtional Include Directories`. Either way, due to NuGet packaging, it must be w.r.t. root of solution.